framework/src/outputs/Checkpoint.C

```diff
@@ -104,6 +108,16 @@ Checkpoint::Checkpoint(const InputParameters & parameters)
   // 'The --output-wall-time-interval parameter is necessary for testing
   // and should only be used in the test suite.
   Output::setWallTimeIntervalFromCommandLineParam();
+  // We want to do this here and not in the constructor so it overrides --output-wall-time-interval
+  if (!getParam<bool>("wall_time_checkpoint"))
+  {
+    _wall_time_interval = std::numeric_limits<Real>::max();
+    mooseInfo("Wall time checkpoints are disabled.");
+  }
+  else
+    mooseInfo(
+        "Checkpoints will be written every " + std::to_string(_wall_time_interval) +
+        " seconds of wall time or at the end of the current timestep, whichever occurs later.");
```

Comment:
Adding a new moose info print to _nearly every run_ isn't a practice we like to do too much. These should be removed

Reply:
Could you please post this comment on #27205?

Reply:
Commented